bin/env.sh

```diff
@@ -17,4 +17,10 @@ export supportRulesengine=nexus3.edgexfoundry.org:10004/docker-support-rulesengi
 export exportClient=nexus3.edgexfoundry.org:10004/docker-export-client-go:1.0.0
 export exportDistro=nexus3.edgexfoundry.org:10004/docker-export-distro-go:1.0.0
 export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual:0.6.0
+
+export vault=nexus3.edgexfoundry.org:10004/docker-edgex-vault:0.3.0
+export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker:0.3.0
```

Comment:
this should be `export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker-go:0.3.0`